ambry-tools/src/main/java/com.github.ambry/tools/perf/rest/NettyPerfClient.java

```diff
@@ -319,34 +350,42 @@ public void initChannel(SocketChannel ch) throws Exception {
     logger.info("NettyPerfClient started");
   }
 
+  private void maybeAddSslHandler(SocketChannel ch) {
```

Comment:
doc

Reply:
just removed this method and moved it to the `ChannelInitializer`